### ISSUE TITLE
AOSP 11.0 v311  nav bar, status bar, soundbar not showing

### ISSUE THREAD

**Issue by turuncu2011** [phhusson/treble_experimentations#1960]
AOSP 11.0 v311, status bar, nav bar and sound panel are still missing. Why don't you explain a solution for this?

**Comment by JonnyTech** [comment]
All ok here, you must have an issue in your settings/data, I recommend a clean flash.

**Comment by phhusson** [comment]
I didn't explain a solution for this, because I wasn't aware of this issue.

**Comment by turuncu2011** [comment]
Can you explain how to make a clean flash?

**Comment by turuncu2011** [comment]
Now you have learned that there is such a problem, what is the solution?

**Comment by phhusson** [comment]
The title of the issue is "AOSP 11.0 v311"
I don't understand how this is an issue, this is just the name of a release.

**Comment by JonnyTech** [comment]
> Can you explain how to make a clean flash?

Flash system + wipe data = clean flash
Flash system, keep data = dirty flash

**Comment by turuncu2011** [comment]
> Sorunun başlığı "AOSP 11.0 v311"
> Bunun nasıl bir sorun olduğunu anlamıyorum, bu sadece bir sürümün adı.

Isn't it a problem that the navigation bar, status bar and sound panel do not appear after installation?

**Comment by phhusson** [comment]
Oh, sure it is, it just wasn't what the title said.

Your aggressiveness towards me also is a problem.

Did you do a clean flash? Which GSI are you testing? What's your device?

**Comment by turuncu2011** [comment]
> Oh, elbette öyle, sadece başlığın söylediği şey değildi.
> 
> Bana karşı saldırganlığın da bir sorun.
> 
> Temiz bir flaş yaptın mı? Hangi GSI'yi test ediyorsunuz? Cihazınız

> Oh, söz konusu, sadece başlığın amacı şeydi.
> 
> Bana karşı saldırganlığın da bir sorun.
> 
> Temiz bir flaş yaptın mı? Hangi GSI'yi test edin? Cihazınız nedir?



> Oh, elbette öyle, sadece başlığın söylediği şey değildi.
> 
> Bana karşı saldırganlığın da bir sorun.
> 
> Temiz bir flaş yaptın mı? Hangi GSI'yi test ediyorsunuz? Cihazınız nedir?


My device is Moto Z, I did a clean flash, it's the same again. I tried system-roar-arm64-ab-gapps.img.xz and others, the result is the same. There was no problem in previous versions.

**Comment by turuncu2011** [comment]
> Moto Z, resmi bir OEM Tiz cihazı değildir. Bunun cihazınızla nasıl ilgili olabileceğinden emin değilim, bu yüzden kontrol edeceğim. Lütfen logcats sağlayın: Önyükleme yapın, 30 saniye bekleyin, `adb logcat -b all -d > log.txt` yapın ve log.txt dosyasını ekleyin

Thank you, I will try your suggestion.

**Comment by phhusson** [comment]
Why did you close it?

**Comment by sukkysounds** [comment]
Hello sir i have the same issue on my infinix x623

**Comment by sukkysounds** [comment]
at first i couldnt boot at all untill i installed phh su and permissiver  and when i could finally boot i couldnt acces stuffs like the status bar and the navbar 
and ill just get a system ui has stopped working error

**Comment by phhusson** [comment]
well I guess @turuncu2011 fixed it, so ask them how they fixed it

**Comment by turuncu2011** [comment]
> @turuncu2011 düzeltmiş sanırım , nasıl düzelttiklerini sorun

No, I couldn't fix it. I didn't find any useful solution, so I installed Havoc OS and there was no problem. So I closed the discussion.

**Comment by turuncu2011** [comment]
I reopened it after seeing people having the same issue.

**Comment by sukkysounds** [comment]
Please phh is there any possible fix or zip file I missed or haven't flashed?
Because I have been able to run previous builds (quack)
 smoothly

**Comment by turuncu2011** [comment]
> Lütfen phh, kaçırdığım veya flashlamadığım olası bir düzeltme veya zip dosyası var mı?
> Çünkü önceki yapıları (şarlatan)
> sorunsuzca çalıştırabildim

I had the same problem, I couldn't find a solution. I installed "Havoc OS- arm64 ab", it worked normally, I recommend it.

**Comment by sukkysounds** [comment]
> > Lütfen phh, kaçırdığım veya flashlamadığım olası bir düzeltme veya zip dosyası var mı?
> > Çünkü önceki yapıları (şarlatan)
> > sorunsuzca çalıştırabildim
> 
> I had the same problem, I couldn't find a solution. I installed "Havoc OS- arm64 ab", it worked normally, I recommend it.

Android R?
